201-front-door-premium-storage-blobs-private-link/modules/storage.bicep

```diff
@@ -0,0 +1,47 @@
+@description('The location into which the Azure Storage resources should be deployed. When using Private Link origins with Front Door Premium during the preview period, there is a limited set of regions available for use. See https://docs.microsoft.com/en-us/azure/frontdoor/standard-premium/concept-private-link#limitations for more details.')
+@allowed([
+  'eastus'
+  'westus2'
+  'southcentralus'
+])
+param location string
+
+@description('The name of the Azure Storage account to create. This must be globally unique.')
+param accountName string
+
+@description('The name of the SKU to use when creating the Azure Storage account.')
+@allowed([
+  'Standard_LRS'
+  'Standard_GRS'
+  'Standard_ZRS'
+  'Premium_LRS'
+])
+param skuName string
+
+@description('The name of the Azure Storage blob container to create.')
+param blobContainerName string
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2019-06-01' = {
+  name: accountName
+  location: location
+  kind: 'StorageV2'
+  sku: {
+    name: skuName
+  }
+  properties: {
+    networkAcls: {
+      defaultAction: 'Deny'
+    }
+  }
+}
+
+resource blobContainer 'Microsoft.Storage/storageAccounts/blobServices/containers@2020-08-01-preview' = {
+  name: '${storageAccount.name}/default/${blobContainerName}'
+  properties:{
+    publicAccess: 'None'
+  }
+}
+
+output blobEndpointHostName string = replace(replace(storageAccount.properties.primaryEndpoints.blob, 'https://', ''), '/', '')
+output storageResourceId string = storageAccount.id
+output storageLocation string = storageAccount.location

```

Comment:
Same as in the other PR - since this has to be passed in, not sure it's needed as an output...

Reply:
Done.